settings.gradle

```diff
@@ -1 +1 @@
-include 'lib:ozwillo-java-spring-integration'
+include 'util:ozwillo-java-spring-integration'
```

Comment:
This file should not exist anymore (now that this lib is managed via the maven repository).